src/Plugins/RunawayProcessKiller/RunawayProcessKillerExtension.cs

```diff
@@ -38,9 +38,11 @@ public RunawayProcessKillerExtension()
             // Default initializer
         }
 
-        public RunawayProcessKillerExtension(String pidfile)
+        public RunawayProcessKillerExtension(String pidfile, int stopTimeoutMs = 5000, bool stopParentFirst = false)
```

Comment:
Yep, commented in the PR. Will fix in the follow-up though it is harmless in the current version (WinSW API is for internal use only)